tests/services/generateFlightsByDate.test.ts

```diff
@@ -1,27 +1,60 @@
 import 'jest';
 import { DateTime } from 'luxon';
+import { flightCache } from '../../src/FlightCache';
 import { generateFlightsByDate } from '../../src/services/generateFlightsByDate';
+import { getMock } from '../testUtils/getMock';
 
-describe('FlightCache ', () => {
-  const dateString = '2021-01-01';
-  const isoDate = DateTime.fromISO(dateString, { zone: 'utc' });
+const mockFlights = [
+  {
+    flightNumber: '4858',
+    origin: { code: 'DFW', city: 'Dallas-Fort Worth', timezone: 'America/Chicago', location: { latitude: 32.8998, longitude: 97.0403 } },
+    destination: { code: 'PHL', city: 'Philadelphia', timezone: 'America/New_York', location: { latitude: 39.8729, longitude: -75.2437 } },
+    distance: 7393,
+    duration: { locale: '19h 3m', hours: 19, minutes: 3 },
+    departureTime: '2020-01-01T03:46:40.000-05:00',
+    arrivalTime: '2020-01-01T23:49:40.000-04:00',
+    aircraft: { model: '738', passengerCapacity: { total: 160, main: 144, first: 16 }, speed: 400 },
+  },
+];
+jest.mock('../../src/FlightCache.ts', () => ({
+  flightCache: {
+    getFlights: jest.fn().mockReturnValue([]),
+    cacheFlights: jest.fn(),
+  },
+}));
 
+describe('generateFlightsByDate ', () => {
   beforeEach(() => {
-    jest.resetModules();
+    jest.resetAllMocks();
   });
 
+  const dateString = '2021-01-01';
+  const isoDate = DateTime.fromISO(dateString, { zone: 'utc' });
+
   it('should generate flights by seed', () => {
     const flights = generateFlightsByDate(isoDate);
     expect(flights.length).toBeGreaterThan(0);
   });
 
-  it('should generate flights from cache when same seed is used', () => {
+  it('should generate flights the same flights twice when using the same date', () => {
+    const flights1 = generateFlightsByDate(isoDate);
+    expect(flights1.length).toBeGreaterThan(0);
+    const flights2 = generateFlightsByDate(isoDate);
+    expect(flights2.length).toBeGreaterThan(0);
+    expect(flights1).toEqual(flights2);
+  });
```

Comment:
We actually don't need to call it twice anymore since it's mocked. We can just mock the cache and expect that the same thing is returned.

```suggestion
  it('should return flights from cache if there are flights', () => {
    getMock(flightCache.cacheFlights).mockReturnValueOnce(mockFlights);

    const flights = generateFlightsByDate(isoDate);
    expect(flights).toBe(mockFlights);
  });
```